scripts/archive_leg_to_csv.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+"""Script to create CSVs of legislators given a state and session"""
+
+import csv
+from collections import Counter
+from utils import get_jurisdiction_id, init_django
+import click
+
+
+init_django()
+
+from opencivicdata.legislative.models import LegislativeSession, Bill, PersonVote  # noqa
+
+
+def archive_leg_to_csv(state_abbr=None, session=None):
+    output_filename = "data/archive_data_legislators/" + state_abbr + session + "legislators.csv"
+
+    jurisdiction_id = get_jurisdiction_id(state_abbr)
+
+    voter_dictionary = Counter()
+
+    bills = Bill.objects.filter(
+        legislative_session__identifier=session,
+        legislative_session__jurisdiction_id=jurisdiction_id,
+    ).values_list("id", flat=True)
+    for bill in bills:
+        voters = PersonVote.objects.filter(vote_event_id__bill_id=bill)
+
+        for voter in voters:
```

Comment:
I just noticed, this considers everyone unmatched, we only need to count someone in this sheet if they don't have a person_id assigned

Reply:
Query now set to:

`voters = PersonVote.objects.filter(vote_event_id__bill_id=bill, voter_id=None)`